source/app.js

```diff
@@ -0,0 +1,64 @@
+import browser from 'detect-browser'
+import translations from './translations'
+
+(function () {
+  let options = INSTALL_OPTIONS
+  const {localStorage} = window
+  const DAY_DURATION = 1000 * 60 * 60 * 24
+  const now = new Date()
+  const weekAgo = new Date(now - DAY_DURATION * 7)
+
+  const language = window.navigator.language || window.navigator.userLanguage || 'en'
+  const [messageLabel, moreLabel] = translations[language] || translations[language.substring(0, 2)] || translations.en
+
+  const appElement = document.createElement('cloudflare-app')
+  appElement.setAttribute('app-id', 'a-better-browser')
+
+  function updateElement () {
+    const browserVersion = parseFloat(browser.version.match(/(\d.)\./))
+    const browserMinimum = options[browser.name] || 0
+
+    const outdated = browserVersion < browserMinimum
+    const seenRecently = localStorage.cfBetterBrowserDismissedAt && weekAgo >= new Date(parseInt(localStorage.cfBetterBrowserDismissedAt, 10))

```

Comment:
It's an interesting problem, does the apps platform allow itself to work on IE6? It's kinda required for an app like this one.